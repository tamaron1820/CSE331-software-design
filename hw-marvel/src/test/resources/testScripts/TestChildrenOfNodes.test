# Test if parent nodes have correct children

# To run this test, the marvel dataset must exist at
#   hw-marvel/src/main/resources/data/soccerPlayer.csv

LoadGraph soccer soccerPlayer.csv

ListChildren soccer Haland
ListChildren soccer Mbappe
ListChildren soccer Messi
ListChildren soccer Neymar
ListChildren soccer Pedri
ListChildren soccer Robinho