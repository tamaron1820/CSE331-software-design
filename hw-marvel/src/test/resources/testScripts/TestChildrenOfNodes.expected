# Test if parent nodes have correct children

# To run this test, the marvel dataset must exist at
#   hw-marvel/src/main/resources/data/soccerPlayer.csv

loaded graph soccer

the children of Haland in soccer are:
the children of Mbappe in soccer are: Messi(Paris-Saint-Germain) Neymar(Paris-Saint-Germain)
the children of Messi in soccer are: Mbappe(Paris-Saint-Germain) Neymar(Barcelona) Neymar(Paris-Saint-Germain) Pedri(Barcelona)
the children of Neymar in soccer are: Mbappe(Paris-Saint-Germain) Messi(Barcelona) Messi(Paris-Saint-Germain) Pedri(Barcelona) Robinho(Santus)
the children of Pedri in soccer are: Messi(Barcelona) Neymar(Barcelona)
the children of Robinho in soccer are: Neymar(Santus)